crates/league-modpkg/src/chunk.rs

```diff
@@ -0,0 +1,58 @@
+use std::{
+    borrow::Cow,
+    io::{BufReader, Read},
+};
+
+use byteorder::{ReadBytesExt as _, LE};
+use io_ext::ReaderExt as _;
+
+use crate::error::ModpkgError;
+
+#[derive(Debug, PartialEq, PartialOrd)]
+pub struct ModpkgChunk {
+    path: Cow<'static, str>,
+    path_hash: u64,
+    compressed_size: usize,
+    uncompressed_size: usize,
+    data_offset: usize,
+    checksum: u64,
+}
```

Comment:
I think this misses:
1. Index into wad lookup table (this is unfortunately necessary because mods can add new files and we need some way to figure in which wad they should go?)
2. Index into feature/tag lookup table (this is what we discussed before about being able to quickly disable parts of mod)

Maybe both of those could be one table (where wad names would be internal "hidden" feature/tag)